crates/external_services/Cargo.toml

```diff
@@ -49,7 +49,8 @@ api_models = { version = "0.1.0", path = "../api_models", optional = true }
 
 
 [build-dependencies]
-tonic-build = "0.12"
+tonic-build = { version =  "0.12" , optional = true }
+router_env = { version = "0.1.0", path = "../router_env", default-features = false  }
```

Comment:
For now, this dependency can also be optional, right? Behind the `dynamic_routing` feature flag perhaps?

Reply:
Just the build dependency right?